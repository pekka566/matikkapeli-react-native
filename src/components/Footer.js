import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';

const Footer = () => {
  return (
    <View
      style={{
        width: '100%',
        alignItems: 'center',
        alignSelf: 'flex-end',
        justifyContent: 'flex-end'
      }}
    >
      <Text>Done with React Native.</Text>
    </View>
  );
};

export default Footer;
