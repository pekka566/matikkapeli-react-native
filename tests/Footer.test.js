import 'react-native';
import React from 'react';
import Footer from '../src/components/Footer';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
