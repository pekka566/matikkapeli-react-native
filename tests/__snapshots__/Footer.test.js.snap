// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "flex-end",
      "justifyContent": "flex-end",
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Done with React Native.
  </Text>
</View>
`;
